prow/config.yaml

```diff
@@ -367,12 +367,15 @@ presubmits:
     trigger: "(?m)^/test pull-kubernetes-e2e-gce-gpu-prow,?(\\s+|$)"
     spec:
       containers:
-      - args:
+      - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170829-93ec4e74 
```

Comment:
keep the latest for now?

Reply:
I'd like to try and see this job stable enough to convert it to being the default at some point soon, I think `latest` should be reserved for canary jobs.

Reply:
:confused: then use the up-to-date image, this tag is from bazelbuild I think

Reply:
Done (I think).